### DISCUSSION TITLE
Login?

### DISCUSSION THREAD

**Discussion by FrancescoXX** [FrancescoXX/rustcrab#43] [Ideas]
My question is: does it make sense to create a sort of login, so people logged can have additional features?

And which technology would you use?

My suggestions are:
- Clerk
- Appwrite

**Comment by bbbjames** [comment]
i might try supabase, or appwrite - i dont know if it makes sense

**Comment by FrancescoXX** [comment]
those are definitively good solutions. maybe they can sponsor the project? not sure. 

Otherwise I will just use something
